src/app.rb

```diff
@@ -75,7 +103,10 @@ def list_all_genres
   end
 
   def list_all_labels
-    puts 'List of labels'
+    puts 'Sorry, there are no labels in the label list' if @label.empty?
+    @label.each do |label|
+      puts "title: #{label.title}, color: #{label.color}"
```

Comment:
I think it would be more useful to the user to see the title of the label instead of the color.
So for example when adding a book you can select Science Fiction instead of red. Don't you think